src/extensions/windows/Microsoft.DotNet.UpgradeAssistant.Extensions.Windows/WinformsDefaultFontUpdater.cs

```diff
@@ -32,9 +38,79 @@ public WinformsDefaultFontUpdater(ILogger<WinformsDefaultFontUpdater> logger)
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
         }
 
-        public Task<IUpdaterResult> ApplyAsync(IUpgradeContext context, ImmutableArray<IProject> inputs, CancellationToken token)
+        private string ProcessAppConfigFile(IProject project)
+        {
+            var hdpiValue = "SystemAware";
+            var appConfigFilePath = project.FindFiles("App.config").FirstOrDefault();
+            ConfigFile appConfigFile;
+            if (File.Exists(appConfigFilePath))
+            {
+                appConfigFile = new ConfigFile(appConfigFilePath);
+                var highDPISetting = appConfigFile.Contents.XPathSelectElement(HighDPIConfiguration);
+                if (highDPISetting is not null)
+                {
+                    var hdpi = highDPISetting.Elements("add").Where(e => e.Attribute("key").Value == "DpiAwareness").FirstOrDefault();
+                    if (hdpi is not null)
+                    {
+                        hdpiValue = hdpi.Attribute("value").Value;
+                        _logger.LogDebug("Found DpiAwareness Setting with {value}", hdpiValue);
+                    }
+                }
+            }
+
+            return hdpiValue;
+        }
+
+        private async Task UpdateHighDPISetting(IProject project)
+        {
+            var programFile = project.FindFiles("Program.cs").FirstOrDefault();
+            if (programFile is not null && File.Exists(programFile))
+            {
+                var programFileContent = File.ReadAllLines(programFile);
+
+                if (!programFileContent.Contains("Application.SetHighDpiMode"))
```

Comment:
Wouldn't `Contains` look for a line that's an exact match for this string? I don't think that's what we want since something like `Application.SetHighDpiMode(HighDpiMode.Foo)` should be considered a match but I suspect wouldn't be since programFileContent wouldn't contain exactly `Application.SetHighDpiMode`.